{{cookiecutter.project_shortname}}/docker/wait-for-services.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+{% include 'misc/header.py' %}
+
+# Verify that all services are running before continuing
+function check_ready() {
+    RETRIES=5
+    while ! $2
+    do
+        echo "Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+{%- if cookiecutter.database == 'postgresql'%}
+_db_check(){ docker-compose exec --user postgres db bash -c "psql -c '\l'" &>/dev/null; }
```

Comment:
If somebody knows a better way to do this ....

Reply:
There is [`pg_isready`](https://www.postgresql.org/docs/9.3/static/app-pg-isready.html), might be useful here

Reply:
I know about this, but it doesn't seem to be installed in the default docker image, I'll look a bit more into it.